taxi/forms.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def license_number_func(license_number):
+    if (
+            len(license_number) != 8
+            or not license_number[0:3].isupper()
+            or not license_number[-5:].isdigit()
```

Comment:
If you want to make sure that the first three characters are strictly uppercase letters (and not symbols or whitespace), you should also check that they are alphabetic characters using .isalpha()